src/gameplay/game.ts

```diff
@@ -1860,11 +1869,22 @@ class Game extends Room {
     );
   }
 
+  updateCloseVoteHistory(closeVoteHistory: boolean) {
+    this.closeVoteHistory = closeVoteHistory;
+
+    this.sendText(
+      this.allSockets,
+      `Close vote history option set to ${closeVoteHistory}.`,
+      'server-text',
+    );
+    
```

Comment:
Nit: can we delete this space? Have you set up prettier and auto-format on save?